zng/docs/spec.md

```diff
@@ -648,15 +588,27 @@ and references the types accordingly in three values;
 1:[hello;world;]
 0:this is a semicolon: \x3b;
 ```
-which represents a stream of the following three values:
+which represents a stream of the three values, that could be expressed in JSON
+as:
 ```
-string("hello, world")
-record(a:"hello",b:"world")
-string("this is a semicolon: ;")
+"hello, world"
+{"a": "hello", "b": "world"}
+"this is a semicolon: ;"
```

Comment:
I've gone ahead and shown these as JSON values and called them out as such. When making a comparison I feel like it's slightly more helpful to speak a lingua franca rather than use a pseudocode syntax.